### PR TITLE
feat: expose a RNTO event

### PR DESCRIPTION
**Pull request by lafin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,52 +69,52 @@ Supported protocols:
 - `ftp` Plain FTP
 - `ftps` Implicit FTP over TLS
 
-_Note:_ The hostname must be the external IP address to accept external connections. `0.0.0.0` will listen on any available hosts for server and passive connections.  
+_Note:_ The hostname must be the external IP address to accept external connections. `0.0.0.0` will listen on any available hosts for server and passive connections.
```
Comment (trs) [inline README.md]: These spaces are actually required for markdown to render the next line _on_ the next line.

With trailing spaces:
image

Without trailing spaces:
image

If you're using VSCode you can change this in `settings.json`:

```
    "files.trimTrailingWhitespace": true,
    "[markdown]": {
        "files.trimTrailingWhitespace": false
    },
```

File: src/connection.js [hunk 2]
```diff
@@ -30,10 +30,9 @@ class FtpConnection extends EventEmitter {
       this.server.emit('client-error', {connection: this, context: 'commandSocket', error: err});
     });
     this.commandSocket.on('data', this._handleData.bind(this));
-    this.commandSocket.on('timeout', () => {
+    this.commandSocket.on('timeout', (err) => {
```
Comment (trs) [inline src/connection.js]: Nice catch 👍

File: README.md (hunk 1 above)
Comment (trs) [reply README.md]: It'd be nice to work on a nicer, simpler readme and better API documentation.
#dreams

**Comment by lafin** [comment]
done

**Comment by trs** [comment]
Awesome, thank you!

**Pull request merged**